packages/credentials/src/issuer.ts

```diff
@@ -15,6 +15,16 @@ import type { CTypeLoader } from './ctype/index.js'
 import type { IssuerOptions, SubmitOverride } from './interfaces.js'
 
 export type { IssuerOptions }
+import * as Kilt from '@kiltprotocol/sdk-js'
+import * as KiltChain from '@kiltprotocol/chain-helpers'
+import type {
+  KiltAddress,
+  SignerInterface,
+  KeyringPair,
+  DidUrl,
+} from '@kiltprotocol/types'
+import {authorizeTx} from "@kiltprotocol/did"
```

Comment:
make sure you run prettier for auto-formatting

Reply:
Fixed at: https://github.com/KILTprotocol/sdk-js/pull/918/commits/ccac9b90e1ceac14a1c26d443d3fcef994745055